test/integration/auth/node_test.go

```diff
@@ -249,6 +252,18 @@ func TestNodeAuthorizer(t *testing.T) {
 		})
 	}
 
+	updatePVCCapacity := func(client clientset.Interface) error {
+		patchBytes := []byte(`{"status":{"capacity": {"storage": "500Gi"}}}`)
+		_, err = client.Core().PersistentVolumeClaims("ns").Patch("mypvc", types.StrategicMergePatchType, patchBytes, "status")
+		return err
+	}
+
+	updatePVCPhase := func(client clientset.Interface) error {
+		patchBytes := []byte(`{"status":{"phase": "Bound"}}`)
+		_, err = client.Core().PersistentVolumeClaims("ns").Patch("mypvc", types.StrategicMergePatchType, patchBytes, "status")
```

Comment:
same here, declare `err` inside the function:
```
_, err := ...
```

Reply:
fixed